crates/context_aware_config/src/api/default_config/types.rs

```diff
@@ -5,6 +5,14 @@ use superposition_types::RegexEnum;
 
 #[derive(Debug, Deserialize)]
 pub struct CreateReq {
+    pub value: Value,
+    pub schema: Map<String, Value>,
```

Comment:
can we validate that schema should not be empty , currently it is not there ,should we allow empty schema , if one dont want type safety ?